sdk/textanalytics/azure-ai-textanalytics/tests/test_text_analytics.py

```diff
@@ -7,7 +7,7 @@
 from azure.core.credentials import AzureKeyCredential
 from azure.ai.textanalytics import TextAnalyticsClient
 from azure.ai.textanalytics import _models
-from azure.ai.textanalytics._generated import models as _generated_models
+from azure.ai.textanalytics._generated.v3_1_preview_1 import models as _generated_models
```

Comment:
just for my own understanding, we only test 3.1-preview.1?

Reply:
for the models repr yes. For all other tests, we mainly test v3.0, but right now I check to see if I can initialize a 3.1-preivew.1 client and that api version is passed correctly to the service https://github.com/Azure/azure-sdk-for-python/blob/8898318b199a09f34645233006b03a3b5446fa00/sdk/textanalytics/azure-ai-textanalytics/tests/test_multiapi.py